pkg/server/config.go

```diff
@@ -2,9 +2,13 @@ package server
 
 import (
 	"flag"
+	"log"
+	"strings"
 
 	"github.com/spf13/pflag"
 
+	kubeoptions "k8s.io/kubernetes/pkg/kubeapiserver/options"
```

Comment:
Isn't this going to be a disallowed import?

Reply:
It's fine for now, given that all of kcp is in the prototyping phase. But if we want to do this long-term/"for real", we need to move anything we need in k/k/pkg into k/k/staging & publish it as a standalone repo (or move it into a preexisting staging dir/repo).